lib/metric_fu/metrics/reek/generator.rb

```diff
@@ -8,21 +10,18 @@ def emit
       files = files_to_analyze
       if files.empty?
         mf_log "Skipping Reek, no files found to analyze"
-        @output = run!([], config_files)
+        run!([], config_files)
```

Comment:
@output was used in analyze where the result from reek is passed to metric_fu structure.

This output content differ between reek versions and therefore requires different parsing to same metric_fu structure.

I moved this parsing and into different ReekExaminer classes, the ReekExaminer.get method will create proper parser depending on Reek gem version.